### PR TITLE
[feat] implement map view for site creation. code cleanup. add previe…

### PR DESCRIPTION
**Pull request by rodocite**
…w to photo upload.

### CODE REVIEW

File: app/config/routes.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import SiteDetail from '../components/SiteDetail';
 import CreateSection from '../components/CreateSection';
 import FilterSection from '../components/FilterSection';
 import SearchSection from '../components/SearchSection';
+import CreateSectionMapView from '../components/CreateSectionMapView';
```
Comment (ha404) [inline app/config/routes.js:12]: Can we rename this one to `CreateLocationSelector`?

File: app/config/routes.js [hunk 2]
```diff
@@ -23,6 +24,7 @@ export default (
     <Route name="create" handler={CreateSection} />
     <Route name="filter" handler={FilterSection} />
     <Route name="search" handler={SearchSection} />
+    <Route name="create-mapview" handler={CreateSectionMapView} />
```
Comment (ha404) [inline app/config/routes.js:27]: Can we rename this handler to `CreateLocationSelector`?

File: app/components/CreateSectionMapView.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  }
+}
+CreateSectionMapView.propTypes = {
+  getSites: React.PropTypes.func.isRequired,
+  sites: React.PropTypes.array.isRequired,
+  onMarkerClick: React.PropTypes.func.isRequired,
+  getLatLng: React.PropTypes.func.isRequired,
+};
+
+CreateSectionMapView.contextTypes = {
```
Comment (ha404) [inline app/components/CreateSectionMapView.jsx:99]: Can we rename this to `CreateLocationSelector`?

File: app/components/CreateSectionMapView.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          return (
+            <Marker {...marker}
+              onClick={this.props.onMarkerClick.bind(this, site)} />
+          );
+        })}
+      </GoogleMap>
+    );
+  }
+}
+CreateSectionMapView.propTypes = {
```
Comment (ha404) [inline app/components/CreateSectionMapView.jsx:92]: Can we rename this to `CreateLocationSelector`?

File: app/components/CreateSectionMapView.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {GoogleMap, Marker, Circle} from 'react-google-maps';
+
+// styles
+import '../styles/components/MapSection';
+export default class CreateSectionMapView extends React.Component {
```
Comment (ha404) [inline app/components/CreateSectionMapView.jsx:6]: Can we rename this to `CreateLocationSelector`?

**Comment by rodocite** [comment]
Renaming file/references and submitting a pull request from a renamed repository.

**Pull request closed without merging**